code/frameworks/angular/src/preset.ts

```diff
@@ -16,7 +17,7 @@ export const previewAnnotations: StorybookConfig['previewAnnotations'] = (
 ) => {
   const annotations = [...entries, require.resolve('./client/config')];
 
-  if (options.configType === 'PRODUCTION') {
+  if ((options as any as StandaloneOptions).enableProdMode) {
```

Comment:
is `options.configType === 'PRODUCTION'` not relevant anymore? Do you need to check for either of these options? or should `enableProdMode` have a default value which would be `options.configType === 'PRODUCTION'`?

Reply:
`options.configType === 'PRODUCTION'` was the first approach, which was not configurable. It gets replaced by the new one.